code/game/objects/items/weapons/weaponry.dm

```diff
@@ -78,22 +77,32 @@ var/highlander_claymores = 0
 /obj/item/weapon/claymore/highlander/New()
 	..()
 	START_PROCESSING(SSobj, src)
-	highlander_claymores++
 
 /obj/item/weapon/claymore/highlander/Destroy()
-	STOP_PROCESSING(SSobj, src)
-	highlander_claymores--
 	if(nuke_disk)
 		nuke_disk.forceMove(get_turf(src))
 		nuke_disk.visible_message("<span class='warning'>The nuke disk is vulnerable!</span>")
 		nuke_disk = null
+	STOP_PROCESSING(SSobj, src)
 	return ..()
 
+/obj/item/weapon/claymore/highlander/process()
+	if(ishuman(loc))
+		var/mob/living/carbon/human/H = loc
+		loc.layer = FLY_LAYER //NO HIDING BEHIND PLANTS FOR YOU, DICKWEED

```

Comment:
The issue I was having was that chairs and plants have the `ABOVE_ALL_MOBS` layer. This was the smallest one I could use that still made sense.